### PR TITLE
Add security.md

### PR DESCRIPTION
**Pull request by bwateratmsft**

### CODE REVIEW

**Comment by bwateratmsft** [comment]
UPDATE: Maybe wait until they take this PR which fixes a typo (which I already fixed locally): https://github.com/microsoft/microsoft.github.io/pull/198

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.4 BLOCK -->
+
+## Security
+
+Microsoft takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations, which include [Microsoft](https://github.com/Microsoft), [Azure](https://github.com/Azure), [DotNet](https://github.com/dotnet), [AspNet](https://github.com/aspnet), [Xamarin](https://github.com/xamarin), and [our GitHub organizations](https://opensource.microsoft.com/).
+
+If you believe you have found a security vulnerability in any Microsoft-owned repository that meets Microsoft's [Microsoft's definition of a security vulnerability](https://docs.microsoft.com/en-us/previous-versions/tn-archive/cc751383(v=technet.10)), please report it to us as described below.
```
Comment (rcjsuen) [inline SECURITY.md]:
```suggestion
If you believe you have found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](https://docs.microsoft.com/en-us/previous-versions/tn-archive/cc751383(v=technet.10)), please report it to us as described below.
```
`Microsoft'` is duplicated twice.

Comment (rcjsuen) [reply SECURITY.md]: I opened microsoft/microsoft.github.io#199 to fix this in the template.

Comment (bwateratmsft) [reply SECURITY.md]: Aha, I have this: https://github.com/microsoft/microsoft.github.io/pull/198

I didn't notice the extra "Microsoft's", I'll put that into that same PR.

Comment (bwateratmsft) [reply SECURITY.md]: Updated https://github.com/microsoft/microsoft.github.io/pull/198 to include both fixes.

**Pull request merged**